### ISSUE TITLE
covRatio in DAP.pdf doesn't match what Pete's code makes

### ISSUE THREAD

**Issue by mobeets** [mobeets/nullSpaceControl#42]
Maybe a data issue? Did he use a different date?

**Comment by mobeets** [comment]
determinant = product of eigenvalues.

**Comment by mobeets** [comment]
well, it's true that the baseline and minimal hypothesis have very tiny eigenvalues, so the determinant just gets super tiny.

but just looking at the covariance matrices for a given kinematic condition, it's clear minimal and baseline are way off. from top left, clockwise: observed, habitual, minimal, baseline:

![covhyps](https://cloud.githubusercontent.com/assets/1677179/12821553/dc2766ca-cb31-11e5-93db-875628e1e97c.png)

**Comment by mobeets** [comment]
http://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-12-222

**Comment by mobeets** [comment]
http://www.itl.nist.gov/div898/software/dataplot/refman1/auxillar/ks2samp.htm

**Comment by mobeets** [comment]
project eigenvectors onto other data and see how variance is explained.

**Comment by mobeets** [comment]
From ["A simple procedure for the comparison of covariance matrices"](http://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-12-222): You get two scores, each describing the error in terms of mismatch in shape and mismatch in orientation.

This describes the ability of one cov's eigenvector to explain the data of the other, and vice versa, relative to how well the cov's eigenvector describes itself.

Big strength I think is in interpretation: just as we have a error metric for the mean, where 0 is the best, now we have an error metric for the covariance, where 0 is the best.

**Comment by mobeets** [comment]
Okay, I've gone back to Pete's original code as pulled from the repo. It took a few adjustments to get right:
- adding scripts like simpleKalmanWhatever, subCellArray, computeAngle, etc. which I had on hand.
- data_directory needed to be set globally so i added a few lines to load it in the right places
- apparent error using subCellArray fixed by a normal indexing

So there was only the one bug, with subCellArray, which I fixed.

But alas! Minimal hypothesis still has a covError of around 10e-3.

I also noticed some code computing a volitionalBasis other than just the row space's intuitive mapping. And some minimal/baseline hypotheses fit in the latent space rather than neural space.
